test/fs.bats

```diff
@@ -206,9 +206,12 @@ teardown() {
     [[ "$output" == linking* ]] || false
 }
 
+@test "fs.is_ellipsis_symlink should detect symlink pointing back to ELLIPSIS_PACKAGES" {
```

Comment:
As commented below, we should keep the original implementation for this one.

The package specific symlink function doesn't seem to have a placeholder yet? I think we can even give it a proper implementation. We test certain functions with a dummy package that get's installed. It contains a symlink you can test against; https://github.com/ellipsis/dot-test/blob/8a3ebed925f252d9adbe0124bb0523a86117ccb8/ellipsis.sh#L21

Not 100% sure if that package is already installed when these functions get tested. Let my know if you need help finding it out if it doesn't work on the first go!

Reply:
See other discussion about the fact of keeping the original behavior or not of this one.

I'm not very comfortable with tests, but after I submitted this PR, I discovered the `dot-test` and the `dot-test2` repositories in the `tests` dir. I'll try to find out how I can take advantage of those to correctly test this feature.

Btw: the file https://github.com/ellipsis/dot-test2/blob/master/ellipsis.sh is an invalid shell script: Either remove the empty `pkg.link` function, either use:
```
pkg.link() {
  :
}
```

I'll try to come up with something :D